### ISSUE TITLE
Image Header / Footer - no repeat - not appearing at all

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#174]
```
This will never work: 
<script type="text/php">
if ( isset($pdf) ) 
{
  $footer = $pdf->open_object();

  $pdf->image("home/production/www/tmpl/ogo.jpg", 'jpg', 50, 10, 143, 46);

  $pdf->close_object();
  $pdf->add_object($footer, "all");
}
</script>


This will print the image only once:
<script type="text/php">
if ( isset($pdf) ) 
{
  $pdf->image("home/production/www/tmpl/ogo.jpg", 'jpg', 50, 10, 143, 46);
  $footer = $pdf->open_object();

  $pdf->image("home/production/www/tmpl/ogo.jpg", 'jpg', 50, 10, 143, 46);

  $pdf->close_object();
  $pdf->add_object($footer, "all");
}
</script>

What version of the product are you using? On what operating system?
* @version 0.5.1.htischer.20090507 
wihth PHP 5.3.2-1ubuntu4.2

Please provide any additional information below.
The image is ok, I can print it outside of the footer. If I put the second 
script at the end of the page before the </body>, I will the image in the last 
page, but not on the others. 
If I put this second script just after the <body>, I will print the image on 
the first page only, but not on the others.

I have the same problem with the png version of this image.

In short: I add this Image manually to a pdf document, but the header/footer 
system refuses to do the job with it.
```

Original issue reported on code.google.com by `vedio...@gmail.com` on 16 Aug 2010 at 3:42

**Comment by GoogleCodeExporter** [comment]
```
After further searches, I found I am not the only one to have this problem...
http://www.dashinteractive.net/dompdf/index.php?v=2189079

http://groups.google.com/group/dompdf/browse_thread/thread/21404e12244ba8bd/6761
22318a39231f?lnk=gst&q=image#676122318a39231f

Etc....

Except the trick to call the image before opening the footer object works only 
for one page (the first or the last, depending where the script is located into 
the HTML)
```

Original comment by `vedio...@gmail.com` on 17 Aug 2010 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem with every versions (dompdf 0.5.1 and DOMPDF 0.6.0 
beta1)
```

Original comment by `vedio...@gmail.com` on 18 Aug 2010 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
I tried a simple example like this one:


<script type="text/php">
if ( isset($pdf) ) 
{
  $footer = $pdf->open_object();

  $pdf->image("www/test/images/goldengate.jpg", 'jpg', 50, 10, 143, 46);

  $pdf->close_object();
  $pdf->add_object($footer, "all");
}
</script>

with an image from dompdf and everything is ok in dompdf 0.6 beta if I put this 
right after <body> tag.
```

Original comment by `fabien.menager` on 27 Aug 2010 at 6:11

**Comment by GoogleCodeExporter** [comment]
```
I still can't dot it, and I locate my script inside the <body>  </body> tag.

How can I help you to reproduce the problem? Do you have a private email so I 
can give you the complete HTML template and join the pictures I use?

Thanks :)

S.L.
```

Original comment by `vedio...@gmail.com` on 6 Sep 2010 at 9:38

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
i have written below code to add header image to every page

$header = $pdf->open_object();
    $pdf->image("logo.jpg", "jpg",10,80,50,400);
$pdf->close_object();
$pdf->add_object($header,"all");

but the image is displayed left aligned.
how to set the left margin of image
as i have tried, the image parameter goes like this
image(Path of image,Type of Image, Margin-top, width,height,"")
i would like to know what does last parameter stands for.? i have set it to 400 
and 40 but there was no difference
```

Original comment by `harnish....@gmail.com` on 1 Mar 2012 at 8:09

**Comment by GoogleCodeExporter** [comment]
```
@harnish.padia have you updated to dompdf 0.6.0? The latest version no longer 
requires that you specify the image type when you call $pdf->image(). So your 
code would be:

$header = $pdf->open_object();
$pdf->image("logo.jpg",10,80,50,400);
$pdf->close_object();
$pdf->add_object($header,"all");

```

Original comment by `eclecticgeek` on 1 Mar 2012 at 11:29

**Comment by GoogleCodeExporter** [comment]
```
Some help me Nothing is working to put a header on my PDF file
```

Original comment by `osugule...@gmail.com` on 18 Jun 2012 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
@osugule123 please post a follow-up to the support group. It helps if you 
include any relevant information about your installation and, if possible, a 
sample HTML document.
```

Original comment by `eclecticgeek` on 19 Jun 2012 at 1:21

**Comment by GoogleCodeExporter** [comment]
```
@vediovis I hope that you have been able to resolve your issue. If you continue 
to have problems please post a follow-up and we'll take another look.
```

Original comment by `eclecticgeek` on 19 Jun 2012 at 1:23
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
how to convert html file to pdf file without header and footer.
```

Original comment by `esubashi...@gmail.com` on 18 Dec 2012 at 9:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:15
- Added labels: **Restrict-AddIssueComment-Commit**
